contrib/guix/guix-codesign

```diff
@@ -91,7 +91,7 @@ fi
 ################
 
 # Default to building for all supported HOSTs (overridable by environment)
-export HOSTS="${HOSTS:-x86_64-w64-mingw32 x86_64-apple-darwin arm64-apple-darwin}"
+export HOSTS="${HOSTS:-x86_64-w64-mingw32 x86_64-apple-darwin20 arm64-apple-darwin20}"
```

Comment:
Why is this change necessary? It appears that Bitcoin Core will remain on `darwin`-only.

Reply:
The suffix darwin20 was created by depends configuration for macOS as zeromq build was failing with that configuration. I changed it to use cmake and now all the builds are working without the suffix"